lib/middlewares/sanitize_headers_middleware.rb

```diff
@@ -0,0 +1,11 @@
+class SanitizeHeadersMiddleware
+  def initialize(app)
+    @app = app
+  end
+
+  def call(env)
+    env["HTTP_ACCEPT"] = "*/*" if env["HTTP_ACCEPT"] =~ /(\.\.|{|})/
```

Comment:
I think we should let rails decide which types are invalid and set the mime type to `text/html` as fallback, like in the `fallback_to_html_format_if_invalid_mime_type` function here: https://guides.rubyonrails.org/configuring.html#config-exceptions-app.